site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -20,7 +35,9 @@ contributes towards a common purpose.
 User interfaces should be minimal and have intent. They can range from a simple icon, such as the [Google
 Mail Checker extension][2] shown on the right, to [overriding][3] an entire page.
```

Comment:
```suggestion
User interfaces should be minimal and have intent. They can range from a simple icon, such as the [Google
Mail Checker extension][2] shown above, to [overriding][3] an entire page.
```

Also, can we update the screenshot of the Google Mail checker image? Chrome's UI has changed since this image was captured.